modules/network-observability-enriched-flows-kafka.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// network_observability/understanding-network-observability-operator.adoc
+
+:_content-type: Procedure
+[id="network-observability-enriched-flows-kafka_{context}"]
+= Export enriched network flow data
+
+You can store network flows in Kafka. 
```

Comment:
`store` is not really suited here I think; maybe `send` or `export` ? Or maybe something like: `You can send network flows to Kafka, so that they can be consumed by any processor or storage that supports Kafka input.` ?